lib/logstash/inputs/kafka.rb

```diff
@@ -4,26 +4,34 @@
 require 'java'
 require 'logstash-input-kafka_jars.rb'
 
-# This input will read events from a Kafka topic. It uses the the newly designed
-# 0.10 version of consumer API provided by Kafka to read messages from the broker.
+# This input will read events from a Kafka topic. It uses the 0.10 version of
+# the consumer API provided by Kafka to read messages from the broker.
 #
 # Here's a compatibility matrix that shows the Kafka client versions that are compatible with each combination
 # of Logstash and the Kafka input plugin: 
 # 
 # [options="header"]
 # |==========================================================
-# |Kafka Client Version |Logstash Version |Plugin Version |Security Features |Why?
-# |0.8       |2.0.0 - 2.x.x   |<3.0.0 | |Legacy, 0.8 is still popular 
-# |0.9       |2.0.0 - 2.3.x   | 3.x.x |SSL |Works with the old Ruby Event API (`event['product']['price'] = 10`)  
-# |0.9       |2.4.x - 5.x.x   | 4.x.x |SSL |Works with the new getter/setter APIs (`event.set('[product][price]', 10)`)
-# |0.10.0.x  |2.4.x - 5.x.x   | 5.x.x |SSL |Not compatible with the <= 0.9 broker
-# |0.10.1.x  |2.4.x - 5.x.x   | 6.x.x |SSL |Not compatible with <= 0.10.0.x broker
+# |Kafka Client Version |Logstash Version |Plugin Version |Why?
+# |0.8       |2.0.0 - 2.x.x   |<3.0.0 |Legacy, 0.8 is still popular 
+# |0.9       |2.0.0 - 2.3.x   | 3.x.x |Works with the old Ruby Event API (`event['product']['price'] = 10`)  
+# |0.9       |2.4.x - 5.x.x   | 4.x.x |Works with the new getter/setter APIs (`event.set('[product][price]', 10)`)
+# |0.10.0.x  |2.4.x - 5.x.x   | 5.x.x |Not compatible with the <= 0.9 broker
+# |0.10.1.0  |2.4.x - 5.x.x   | 6.0.0 - 6.2.1 |Not compatible with <= 0.10.0.x broker
+# |0.10.1.1  |2.4.x - 5.x.x   | 6.2.2 - 6.x.x |Not compatible with <= 0.10.1.0 broker
```

Comment:
@suyog I see. I had deduced that last line from this PR https://github.com/logstash-plugins/logstash-input-kafka/pull/160. 0.10.1.1 compatibility being enabled in 6.2.2.